src/assets/js/dynamic-addon-cards.js

```diff
@@ -73,6 +72,12 @@
   };
 
   const updateAddonCard = async (card, { parsedUserAgent }) => {
+    const baseURLFromBody = document.querySelector('body').dataset.amoBaseUrl;
```

Comment:
I suppose it is fine here but this could be done only once at the bottom of this file, and we would pass `baseURL` or `baseURLFromBody` as an optional argument (like `parsedUserAgent`).

Reply:
If I do that then the tests I wrote no longer work, because they are designed to test reading the `baseURL` from the body, but with this design we actually pass the `baseURL` into the call to `updateAddonCard`, so we would need a different set of tests. I actually moved this code from the top of this file into `updateAddonCard` in order to be able to get these tests to work. Does that seem ok?